tests/modules/nf-core/canu/test.yml

```diff
@@ -0,0 +1,11 @@
+- name: canu test_hicanu
+  command: nextflow run ./tests/modules/nf-core/canu -entry test_hicanu -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/canu/nextflow.config
+  tags:
+    - canu
+  files:
+    - path: output/canu/test.contigs.fasta
```

Comment:
I saw you had here some `contains:` but removed it again. Why is that? Is the test data not working as expected?

Reply:
Yes, thats a common problem with assemblers.